resolwe_bio/tools/fastqc.sh

```diff
@@ -2,15 +2,15 @@
 
 source /etc/profile.d/resolwe-base.sh
 
-if [[ $# -lt 1 || $# -gt 4 ]]; then
-  re-error "Usage: fastqc.sh reads.fastq.gz [directory [fastqc_archive [fastqc_url]]]"
+if [[ $# -lt 1 || $# -gt 5 ]]; then
+  re-error "Usage: fastqc.sh reads.fastq.gz [directory [fastqc_archive [fastqc_url]]] [fastqc-options]"
```

Comment:
Change the error message to
`Usage: fastqc.sh reads.fastq.gz [directory [fastqc_archive [fastqc_url [fastqc-options...]]]]`.